city_scrapers/spiders/cook_county_board_ethics.py

```diff
@@ -0,0 +1,136 @@
+    timezone = 'America/Chicago'
+    allowed_domains = ['www.cookcountyil.gov']
+
+    def start_requests(self):
+        today = datetime.now()
+        next_month = today.replace(day=28) + timedelta(days=5)
+        for dt in [today, next_month]:
+            mo_str = dt.strftime('%Y-%m')
+            url = 'https://www.cookcountyil.gov/calendar-node-field-date/month/{}'.format(mo_str)
+            yield scrapy.Request(url=url, method='GET', callback=self.parse)
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for url in self._get_event_urls(response):
+            yield scrapy.Request(url, callback=self._parse_event, dont_filter=True)
+
+    def _parse_event(self, response):
+        """Parse the event page."""
+        title = self._parse_title(response)
+        meeting = Meeting(
+            title=title,
+            description=self._parse_description(response),
+            classification=BOARD,
+            start=self._parse_start(response),
+            end=self._parse_end(response),
+            time_notes='',
+            all_day=self._parse_all_day(response),
+            location=self._parse_location(response),
+            links=self._parse_links(response),
+            source=response.url,
+        )
+        meeting['id'] = self._get_id(meeting)
+        meeting['status'] = self._get_status(meeting)
+        return meeting
+
+    def _get_event_urls(self, response):
+        """
+        Get urls for all Board of ethics meetings on the page.
+        """
+        return [
+            response.urljoin(href)
+            for href in response.xpath('//a[contains(text(), "Board of Ethics")]'
+                                       ).css('a::attr(href)').extract()
+        ]
+
+    def _parse_location(self, response):
+        """
+        Parse or generate location. Url, latitude and longitude are all
+        optional and may be more trouble than they're worth to collect.
+        """
+        address = response.xpath('//div[@class="field event-location"]/descendant::*/text()'
+                                 ).extract()
+        address = ' '.join([x.strip() for x in address])
+        address = address.replace('Location:', '').strip()
+        return {
+            'address': address,
+            'name': '',
+        }
+
+    def _parse_all_day(self, response):
+        """
+        Parse or generate all-day status. Defaults to false.
+        """
+        date = response.xpath('//span[@class="date-display-single"]/descendant-or-self::*/text()'
+                              ).extract()
+        date = ''.join(date).upper()
+        return 'ALL DAY' in date
+
+    def _parse_title(self, response):
+        """Parse or generate event title."""
+        return response.xpath('//h1/text()').extract_first()
```

Comment:
Any changes in the title are likely unneeded noise, so we should check for things like "Special" in the title and return the title if it's there, but otherwise just confirm that "Board" is in the title and return "Board of Ethics"

Reply:
If possible please elaborate on things to check for besides "Special".
Can I just return the title if it's not 'Cook County: Board of Ethics'?

Reply:
We don't want any boilerplate like "Cook County" included, so we can basically do that as long as we also remove "Cook County: " (removing it whether there's a colon or not)